.github/workflows/submit.yml

```diff
@@ -83,8 +83,8 @@ jobs:
         run: |
           set -x
           sudo apt-get update
-          sudo apt-get install libgl1-mesa-dev libx11-dev libxxf86vm-dev libxt-dev pkg-config libgtk2.0-dev libgtk-3-dev libxtst-dev gcc-10=10.3.0-1ubuntu1~20.04 g++-10=10.3.0-1ubuntu1~20.04
-          sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 100 --slave /usr/bin/g++ g++ /usr/bin/g++-10
+          sudo apt-get install libgl1-mesa-dev libx11-dev libxxf86vm-dev libxt-dev pkg-config libgtk2.0-dev libgtk-3-dev libxtst-dev gcc-11 g++-11
```

Comment:
Did this install 11.2? The CI is using Focal, and afaik there is no gcc-11.2 for focal?

Reply:
No, it will use whatever the latest is for GitHub actions. There doesn't seem to be an 11.2 package for GHA. It should be sufficient for most testing, but highlights one of the limitations of relying on GitHub Actions for anything more than a sanity test.

Reply:
(based on the printout I see in the log files, the current version of gcc 1.x available on the GHA Linux nodes is 11.1)

Reply:
11.1 is what I'm using on 18.04 and 20.04 as well. I wonder if that is good enough, or are there particular reasons why you want to update to 11.2 ?